README.md

```diff
@@ -309,43 +307,6 @@ skate.define('my-component', {
 
 
 
-#### `events`
-
-Event listeners to add to the custom element. These get bound after the `prototype` is set up and before `created` is called.
-
-```js
-skate.define('my-component', {
-  events: {
-    click (elem, eventObject) {}
-  }
-});
-```
-
-The arguments passed to the handler are:
-
-- `elem` is the DOM element
-- `eventObject` is the native event object that was dispatched on the DOM element
-
-
-
-##### Event Delegation
-
-Event descriptors can use selectors to target descendants using event delegation.
-
-```js
-skate.define('my-component', {
-  events: {
-    'click button' (elem, eventObject) {}
-  }
-});
-```
-
-Instead of firing for every click on the component element - or that bubbles to the component element - it will only fire if a descendant `<button>` was clicked.
-
-Event delegation works with or without a shadow root as it will inspect the event `path` if it exists.
-
-
-
```

Comment:
It's [here](https://github.com/skatejs/skatejs#attrson), but I'll add a section called "Binding Events" that describes it and links off to that as well as provides workarounds for the host element.